src/com/example/supabase.cljs

```diff
@@ -24,3 +26,28 @@
       .-auth
       .user
       (js->clj :keywordize-keys true)))
+
+;; .from('cities')
+;;   .insert([
+;;     { name: 'The Shire', country_id: 554 },
+;;     { name: 'Rohan', country_id: 555 },
+;;   ])
+(defn insert-response [{:keys [response/total user/id]}]
+  (go
+    (try
+      (<p! (-> @client
```

Comment:
this macro is like `<!` but will throw an exception if the promise returns an error.